src/main/java/core/basesyntax/enums/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.enums;
+
+public enum Colors {
```

Comment:
```suggestion
public enum Color {
```

Reply:
I had your review before in another exercise, and you made enum "Color" to "Colors", now now the opposite, so how is right?